internal/sign/job/signer.go

```diff
@@ -117,13 +121,64 @@ func (s *signer) MakeJobTemplate(
 		args = append(args, "--skip-tls-verify-pull")
 	}
 
+	clusterCACM, err := s.caHelper.GetClusterCA(ctx, mod.Namespace)
+	if err != nil {
+		return nil, fmt.Errorf("could not get the cluster CA ConfigMap: %v", err)
+	}
+
+	servingCACM, err := s.caHelper.GetServiceCA(ctx, mod.Namespace)
+	if err != nil {
+		return nil, fmt.Errorf("could not get the serving CA ConfigMap: %v", err)
+	}
+
+	const (
+		trustedCAMountPath  = "/etc/pki/ca-trust/extracted/pem"
+		trustedCAVolumeName = "trusted-ca"
+	)
+
 	volumes := []v1.Volume{
 		utils.MakeSecretVolume(signConfig.KeySecret, "key", "key.priv"),
 		utils.MakeSecretVolume(signConfig.CertSecret, "cert", "public.der"),
+		{
+			Name: trustedCAVolumeName,
+			VolumeSource: v1.VolumeSource{
+				Projected: &v1.ProjectedVolumeSource{
+					Sources: []v1.VolumeProjection{
+						{
+							ConfigMap: &v1.ConfigMapProjection{
+								LocalObjectReference: v1.LocalObjectReference{Name: clusterCACM.Name},
+								Items: []v1.KeyToPath{
+									{
+										Key:  clusterCACM.KeyName,
+										Path: "tls-ca-bundle.pem",
+									},
+								},
+							},
+						},
+						{
+							ConfigMap: &v1.ConfigMapProjection{
+								LocalObjectReference: v1.LocalObjectReference{Name: servingCACM.Name},
+								Items: []v1.KeyToPath{
+									{
+										Key:  servingCACM.KeyName,
+										Path: "ocp-service-ca-bundle.pem",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
```

Comment:
This is quite a specific volume type though. So far that's the only place in the code where we are using it. Does it deserve a function?

Reply:
I think it would make the job creation code cleaner to factor it out into a separate function, but coupled with the comment above, and the typecasting I was having to do for the SA secrets, we probably need to revisit the volume_helper code entirely,  so lets worry not worry about it in this PR.